README.rst

```diff
@@ -486,12 +491,26 @@ Assuming you have ``>=python3.8`` installed, navigate to the directory where you
     pre-commit install &&
     python -m unittest
 
-In case you want to run a single unittest for a newly developed scraper
-
+#### Windows:
 .. code:: shell
 
-    python -m unittest -k <test_file_name>
+    git clone git@github.com:hhursev/recipe-scrapers.git &&
+    cd recipe-scrapers &&
+    python -m venv .venv &&
+    .venv\Scripts\activate &&
+    python -m pip install --upgrade pip &&
+    pip install -r requirements-dev.txt &&
+    pip install pre-commit &&
+    pre-commit install &&
+    python -m unittest
```

Comment:
I don't use Windows myself, but I think that the [Python Launcher](https://docs.python.org/3.12/using/windows.html#launcher) is the preferred way to run Python on Windows.

```suggestion
    git clone git@github.com:hhursev/recipe-scrapers.git &&
    cd recipe-scrapers &&
    py -m venv .venv &&
    .venv\Scripts\activate &&
    python -m pip install --upgrade pip &&
    pip install -r requirements-dev.txt &&
    pip install pre-commit &&
    pre-commit install &&
    python -m unittest
```